tensorflow/lite/tools/versioning/op_version.cc

```diff
@@ -858,30 +857,54 @@ OpSignature GetOpSignature(const OperatorCode* op_code, const Operator* op,
   return op_sig;
 }
 
-void UpdateOpVersion(uint8_t* model_buffer_pointer) {
-  auto model = GetMutableModel(model_buffer_pointer);
+void UpdateOpVersion(flatbuffers::FlatBufferBuilder& builder) {
```

Comment:
@terryheo 
Please check:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/kernels/test_util.cc#L193
The kernel tests need this function to update the op's version.
All kernel test's default op version is 0. If we don't update the op version, the interpreter can't not find the proper op.

Reply:
> I see. We used to have a rule to invoke flatc but removed it since it didn't work well on cross compilation.
> Let me find a way to bundle mutable/schema_generated.h file.

@terryheo 
Could we also commit the mutable/schema_generated.h just as the immutable one in repo? When I check the log of immutable schema header, I saw it's be updated if the schema changes. How about just upload both the immutable and mutable headers?